README.md

```diff
@@ -5,10 +5,64 @@
 
 ## Installation
 
+
 ```
 yarn add --dev jest-styled-components
 ```
 
+This package must be used with either `react-test-renderer` or `enzyme` and `enzyme-to-json` to generate a serialized `tree` from your component.
+
+**Using react-test-renderer**
+
+
+Installation:
+
+```
+yarn add --dev react-test-renderer
+```
+
+Usage:
+
+```js
+import renderer from 'react-test-renderer'
+
+// somewhere in your tests
+it('matches the styled components snapshot', () => {
+  const tree = renderer.create(<MyComponent />).toJSON()
+  expect(tree).toMatchStyledComponentsSnapshot()
+  expect(tree).toHaveStyleRule('display', 'block')
+})
+```
+
+For more info on using react-test-renderer with jest for snapshots see [this](https://facebook.github.io/jest/docs/snapshot-testing.html)
```

Comment:
Can we rephrase this to "Lear more about [Snapshot Testing](https://facebook.github.io/jest/docs/snapshot-testing.html) with Jest." or something similar?

Also, I [personally wrote](https://github.com/facebook/jest/pull/3425) the FAQs in that page :)